code.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "neic-traveltime",
```

Comment:
This is the code.json for neic-traveltime, not neic-locator  there is a pending merge request in with the proper code.json file.